tests/apidocs/util.py

```diff
@@ -22,9 +23,25 @@ def create_schema(self):
             return create_spec(data)
 
     def validate_schema(self, request, response):
+        assert 200 <= response.status_code < 300, response.status_code
+
+        response["Content-Type"] = "application/json"
```

Comment:
is this going to be true for the multi part requests?

Reply:
Yeah openapi-core was breaking because `content-type` sometimes existed but not `Content-Type`. We can add more cases if this breaks any tests.